core/src/test/java/org/frankframework/stream/MessageTest.java

```diff
@@ -1331,32 +1325,30 @@ private void createMessageInShortLivedScope() {
 
 	@Test
 	void testMessageAsStringDoesNotCloseMessage() throws IOException {
-		// Arrange: make it an object, so method can do instanceof check
-		Object msg = Message.asMessage(new StringReader("text"));
+		// Arrange
+		String text = "text";
+		Message msg = Message.asMessage(new StringReader(text));
 
 		// Act
-		String content = Message.asString(msg);
+		String content = msg.asString();
 
 		// Assert
-		Message message = (Message) msg;
-		assertEquals("text", message.asString());
-		assertEquals("text", content);
-		message.close();
+		assertEquals(text, content);
+		msg.close();
 	}
```

Comment:
Deze test is meer bedoeld voor MessageUtil, dat wanneer met MessageUtil.asString doet, de input niet gesloten wordt.

Reply:
Maar deze roept toch niet de static method aan?